core/src/main/scala/scalaz/syntax/std/EitherOps.scala

```diff
@@ -6,7 +6,10 @@ final class EitherOps[A, B](private val self: Either[A, B]) extends AnyVal {
 
   final def disjunction: A \/ B = \/ fromEither self
```

Comment:
Shouldn't this one also be `toDisjunction`?

Reply:
Great observation! I added it to newest version of this PR.